requester/print.go

```diff
@@ -32,8 +32,15 @@ type report struct {
 	average  float64
 	rps      float64
 
+	avgConn  float64
+	avgDns   float64
+	avgReq   float64
+	avgRes   float64
+	avgDelay float64
+
 	results chan *result
 	total   time.Duration
+	trace   bool
```

Comment:
I was thinking of making a separate struct type for  trace report, incase in future if we add extra fields like fastestDns, slowestDns etc. 
Rest of the mentioned changes above and below would be fixed soon